src/test/java/com/google/api/codegen/testdata/discoveries/foo.v1.json.overrides

```diff
@@ -4,5 +4,27 @@
   },
   "js|python": {
     "discoveryDocUrl": "localhost:8080/$discovery/foo?version=v1"
+  },
+  "csharp|go|java": {
+    "methods": {
+      "foo.bar.get": {
+        "fields": {
+          "foobar": {
+            "defaultValue": "\" \""
```

Comment:
I hadn't ruled it out, but now that I think about it... that doesn't seem worth supporting.

The change you're suggesting is non-trivial, I'll follow up with it later if that's alright. I just want this for my tests. I'll open an issue to track.